Sprint-3/implement/get-angle-type.test.js

```diff
@@ -0,0 +1,88 @@
+// Implement a function getAngleType
+
+// Acceptance criteria:
+
+// Given an angle in degrees,
+// When the function getAngleType is called with this angle,
+// Then it should:
+
+// Identify Right Angles:
+// When the angle is exactly 90 degrees,
+// Then the function should return "Right angle"
+
+// Identify Acute Angles:
+// When the angle is less than 90 degrees,
+// Then the function should return "Acute angle"
+
+// Identify Obtuse Angles:
+// When the angle is greater than 90 degrees and less than 180 degrees,
+// Then the function should return "Obtuse angle"fed
+
+// Identify Straight Angles:
+// When the angle is exactly 180 degrees,
+// Then the function should return "Straight angle"
+
+// Identify Reflex Angles:
+// When the angle is greater than 180 degrees and less than 360 degrees,
+// Then the function should return "Reflex angle"
+
+function getAngleType(angle) {
+    if (angle === 90) {
+        return "Right angle";
+    } else if (angle < 90) {
```

Comment:
Nice! 

If you were to revise this function, how could you make it simpler and easier to read?

Reply:
Probably this version of the function is easier to read.
Thank you.

function getAngleType(angle) {
    if (angle === 90) return "Right angle";
    if (angle < 90) return "Acute angle";
    if (angle === 180) return "Straight angle";
    if (angle < 180) return "Obtuse angle";
    if (angle < 360) return "Reflex angle";
    return "Invalid angle";
}